src/python-bindings.lisp

```diff
@@ -56,12 +60,13 @@
                                       :library-name library-name))
                          :collect (coerce-to-c-name (prefix-name (api-function-prefix api) name)))))))
 
-(defun build-python-bindings (library directory &key (omit-init-call nil) (text-between-header-and-exports ""))
+(defun build-python-bindings (library directory &key (omit-init-call nil) (text-between-header-and-exports "")
+                                                     (library-path nil))
   (let ((file-name (concatenate 'string (library-c-name library) ".py")))    
     (with-open-file (stream (merge-pathnames file-name directory)
                             :direction :output
                             :if-exists :supersede)
-      (funcall 'write-default-python-header library stream omit-init-call)
+      (funcall 'write-default-python-header library stream omit-init-call library-path)
```

Comment:
(I realize this was already there, but is funcall doing something special here?)

Reply:
The funcall exists to allow users to patch the FDEFINITION of WRITE-DEFAULT-PYTHON-HEADER to make it do something else.